brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaPlanToSpecTransformer.java

```diff
@@ -82,8 +87,16 @@
     private Alien4CloudToscaPlatform platform;
     private final AtomicBoolean alienInitialised = new AtomicBoolean();
 
+    static {
+        BrooklynFeatureEnablement.setDefault(FEATURE_TOSCA_ENABLED, true);
```

Comment:
Does this require a patched Brooklyn to compile? I get a compile error because `setDefault` is protected.